2015-Fall/adrianbrink/Calculator/Calculator/ViewController.swift

```diff
@@ -0,0 +1,73 @@
+//
+//  ViewController.swift
+//  Calculator
+//
+//  Created by aevitas on 01/09/15.
+//  Copyright (c) 2015 aevitas. All rights reserved.
+//
+
+import UIKit
+
+class ViewController: UIViewController {
+    
+    @IBOutlet weak var display: UILabel!
+    
+    var userIsTyping: Bool = false
+    var lastAction: Bool?
```

Comment:
An optional Bool is a bit of a strange thing. A boolean is a value indicating if a predicate holds or not. It can have two values, true or false. An optional boolean can have three values, true, false, or nil. Then it's not really a boolean anymore.

I would consider initialising this as false.